main/src/library/Channel.flix

```diff
@@ -20,28 +20,28 @@ mod Channel {
     ///
```

Comment:
@magnus-madsen In this file is the typing change clearest - all the other code is just consequences

Reply:
Thanks 👍